setup.py

```diff
@@ -76,7 +76,10 @@
         'ufolint',
         'unidecode',
         'beziers',
-        'cmarkgfm'
+        'cmarkgfm',
+        'vharfbuzz',
```

Comment:
Thoughts about making the new dependencies `extras_requires` packages so that they are optional?  How well does vharfbuzz and anything else new that might be compiled here play with cross-platform installations of fontbakery?

Reply:
My plan was to cut a new fontbakery release after merging this PR. But your comment makes me want to better evaluate the cross-platform support of these new dependencies before committing to a new release.